src/ui/public/notify/notifier.js

```diff
@@ -310,48 +320,69 @@ Notifier.prototype._showFatal = function (err) {
  * @param  {Error|String} err
  * @param  {Function} cb
  */
-Notifier.prototype.error = function (err, cb) {
-  return add({
+Notifier.prototype.error = function (err, opts, cb) {
+  const overrideable = ['lifetime', 'icon'];
+  if (_.isFunction(opts)) {
+    cb = opts;
+    opts = {};
+  }
+
+  const config = _.assign({
     type: 'danger',
     content: formatMsg(err, this.from),
     icon: 'warning',
     title: 'Error',
     lifetime: Notifier.config.errorLifetime,
     actions: ['report', 'accept'],
     stack: formatStack(err)
-  }, cb);
+  }, _.pick(opts, overrideable));
```

Comment:
I think we can use ES2015 here, and inline the `overrideable` variable (which seems a bit confusing as a separate variable):

```javascript
const config = Object.assign({
  type: 'danger',
  content: formatMsg(err, this.from),
  icon: 'warning',
  title: 'Error',
  lifetime: Notifier.config.errorLifetime,
  actions: ['report', 'accept'],
  stack: formatStack(err)
}, _.pick(opts, ['lifetime', 'icon']));
```